src/components/VmDetails/cards/DetailsCard/CloudInit/SysprepForm.js

```diff
@@ -1,19 +1,20 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import {
-  FormControl,
+  Checkbox,
   ControlLabel,
+  FormControl,
   FormGroup,
 } from 'patternfly-react'
 import { msg } from '_/intl'
 import SelectBox from '../../../../SelectBox'
 import timezones from '_/components/utils/timezones.json'
 
-const SysprepForm = ({ idPrefix, vm, onChange }) => {
+const SysprepForm = ({ idPrefix, vm, onChange, enableSysPrepTimezone, selectedSysPrepTimezone }) => {
   const cloudInitHostName = vm.getIn(['cloudInit', 'hostName'])
   const cloudInitPassword = vm.getIn(['cloudInit', 'password'])
-  const cloudInitTimezone = vm.getIn(['cloudInit', 'timezone']) || timezones[0].id
```

Comment:
No one renamed `cloudInitTimezone`. It was removed and `selectedSysPrepTimezone` was used instead. But yeah, it may look like it was renamed, especially [here](https://github.com/oVirt/ovirt-web-ui/pull/1288/commits/4259ad7d512b01a2ed441df5f028310ae920d63a#diff-7d57d8c4e12b6e312c3dcac3620c10171bcf1a5b1e984cb502af0f072e61ba97R57). But don't worry, these changes will be changed anyway.